engine/gamesys/src/dmsdk/gamesys/resources/res_gui.h

```diff
@@ -32,29 +32,27 @@ namespace dmParticle
 namespace dmGameSystem
 {
     struct MaterialResource;
-    struct TextureResource;
-    struct TextureSetResource;
 
     struct GuiSceneTextureSetResource
     {
-        TextureSetResource* m_TextureSet;
-        TextureResource*    m_Texture;
+        void*   m_Resource;
+        uint8_t m_ResourceIsTextureSet : 1;
     };
```

Comment:
this is a dmsdk change, but nobody should be using these structs anyway. Perhaps we shouldn't even expose the struct here.